modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -131,7 +141,8 @@ export default {
   border-width: 0;
   padding: 0;
 
-  [disable]:hover, [disabled] ~ .apos-choice-label-text:hover {
+  [disable]:hover,
+  [disabled] ~ .apos-choice-label-text:hover {
```

Comment:
What does the `~`, is it an equivalent of `>` or `+`?

Reply:
`~`: Sibling element selector
`>`: Direct descendant selector
`+`: Selects the second element when preceded by the first